Modules/posixmodule.c

```diff
@@ -7237,26 +7237,26 @@ os_symlink_impl(PyObject *module, path_t *src, path_t *dst,
     result = CreateSymbolicLinkW(dst->wide, src->wide, flags);
     Py_END_ALLOW_THREADS
 
-    if (windows_has_symlink_unprivileged_flag && !result) {
+    if (windows_has_symlink_unprivileged_flag && !result && ERROR_INVALID_PARAMETER == GetLastError()) {
```

Comment:
Please follow [PEP 7](https://www.python.org/dev/peps/pep-0007) (lines should be no longer than 79 chars).

Reply:
Done, and tests all green. Thanks!